Makefile

```diff
@@ -101,7 +101,7 @@ KUBE_APISERVER=$(TOOLS_BIN_DIR)/kube-apiserver
 ETCD=$(TOOLS_BIN_DIR)/etcd
 
 # Define Docker related variables. Releases should modify and double check these vars.
-REGISTRY ?= gcr.io/$(shell gcloud config get-value project)
+REGISTRY ?= registry.local/fake
```

Comment:
I assume this is just a placeholder, so we don't try to shell out to `gcloud`?

Reply:
correct